Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -8,7 +8,6 @@
 #include <LibJS/Runtime/Array.h>
 #include <LibJS/Runtime/GlobalObject.h>
 #include <LibJS/Runtime/IteratorOperations.h>
-#include <LibJS/Runtime/Temporal/AbstractOperations.h>
```

Comment:
I didn't check every diff, but this header is used by every file it was removed from that I did check - see `get_options_object`